client/v2/autocli/query_test.go

```diff
@@ -121,6 +130,20 @@ func TestCoin(t *testing.T) {
 	assert.DeepEqual(t, conn.lastRequest, conn.lastResponse.(*testpb.EchoResponse).Request, protocmp.Transform())
 }
 
+func TestMap(t *testing.T) {
+	conn := testExecCommon(t, buildModuleQueryCommand,
+		"echo",
+		"1",
+		"abc",
+		"1234foo",
+		"4321bar",
+		"--map-string-string", "foo=bar",
+		"--map-string-uint32", "bar=123",
+		"--map-string-coin", "baz=100000foo",
```

Comment:
can we add a test with multiple coins? so we know it is well supported?

Reply:
added some

Reply:
Right, but are multiple values are supported? Now you've called the flag multiple times, but I meant testing `--map-string-coin baz=1000uatom,foo=1000stake`.
Why is map-string-string deleted from the tests btw?

Reply:
Done